src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerResolverTest.php

```diff
@@ -24,6 +24,24 @@
 
 class ControllerResolverTest extends ContainerControllerResolverTest
 {
+    public function testGetControllerWithServiceWithoutMethod()
+    {
+        class_exists(AbstractControllerTest::class);
```

Comment:
What is the reason for the `class_exists()` call here?

Reply:
No specific reason. Think it's better to remove it but since I have to start from master branch, I should rework the entire new code